src/Bicep.Core/Semantics/ParameterAssignmentSymbol.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using Bicep.Core.Syntax;
+using Bicep.Core.TypeSystem;
+
+namespace Bicep.Core.Semantics
+{
+    public class ParameterAssignmentSymbol : BindableSymbol
+    {
+
+        public ParameterAssignmentSymbol(string name, SyntaxBase declaringSyntax, IdentifierSyntax nameSyntax)
+            : base(name, nameSyntax)
+        {
+            this.DeclaringSyntax = declaringSyntax;
+        }
+        /// <summary>
+        /// Gets the syntax node that declared this symbol.
+        /// </summary>
+        public SyntaxBase DeclaringSyntax { get; }
+
+        /// <summary>
+        /// Gets the type of the symbol.
+        /// </summary>
+        public TypeSymbol Type => throw new NotImplementedException(); // TODO
+
+        public override SymbolKind Kind => throw new NotImplementedException(); // TODO
+
+        public override void Accept(SymbolVisitor visitor)
+        {
+            throw new NotImplementedException(); // TODO
```

Comment:
Let's implement the method since it's just a few lines of code (it's implemented for `ParamFileSymbol`).